src/applications/applicationList/search/option/index.tsx

```
@@ -1,24 +1,23 @@
-}
+  list: string[];
+  onChange: Dispatch<SetStateAction<string>>;
+};
+const Option = ({ list, onChange }: OptionsProps) => {
+  return (
+    <_.option_set>
+      <_.black>
+        <_.white>
+          <_.dark>
+            {list.map((item) => (
+              <_.options onClick={() => onChange(item)}>{item}</_.options>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)